backend/utils/lot_utils.py

```
@@ -0,0 +1,166 @@
+    if existing:
+        # If it exists, increment sequence and try again
+        current_sequence += 1
+        child_lot_number = generate_child_lot_number(parent_chemical.lot_number, current_sequence)
+    
+    # Increment the parent's lot_sequence counter
+    parent_chemical.lot_sequence = current_sequence + 1
+    
+    return child_lot_number, current_sequence
+
+
+def create_child_chemical(parent_chemical, quantity, destination_warehouse_id=None, destination_kit_id=None):
```

Ruff (0.14.0)

Unused function argument: `destination_kit_id`

(ARG001)